advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx

```diff
@@ -39,10 +39,10 @@ All of these components are available on the EDB Postgres AI Cloud Service, and
 Filtering out the data noise and revealing insights and value, Lakehouse analytics brings both structured relational data in Postgres and unstructured data in object storage together for exploration.
 
 * **[Lakehouse nodes](/edb-postgres-ai/analytics/lakehouse)**
-  * At the heart of Analytics is a custom built store for this data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring your data in from the cold, ready for analysis. 
+  * At the heart of Analytics is a custom built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in ready for analysis. 
 
 ## [EDB Postgres AI AI/ML](/edb-postgres-ai/ai-ml)
-* Postgres has proven its capability as a flexible data environment, and Vector data, the core of generative AI, is already infused into EDB Postgres AI providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the pgai extension.
+* Postgres has proven its capability as a flexible data environment, and with Vector data as the core of generative AI, is already infused into EDB Postgres AI providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.
```

Comment:
```suggestion
* Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, ir's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres pgai extension.
```

Reply:
Please check if this is correct. It was hard to follow.

Reply:
```suggestion
* Postgres has proven its capability as a flexible data environment and EDB Postgres AI is already infused with Vector data support, enabling generative AI and providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.
```